lib/rbac/filterer.rb

```diff
@@ -656,7 +656,9 @@ def apply_scope(klass, scope)
     end
 
     def apply_select(klass, scope, extra_cols)
-      scope.select(scope.select_values.blank? ? klass.arel_table[Arel.star] : nil).select(extra_cols)
+      scope = scope.select(klass.arel_table[Arel.star]) if scope.select_values.blank?
```

Comment:
We totally need to add to AR something like `scope.select_also()`.
I'm pretty sure matthewd would merge.

Reply:
Yeah, I could see that being useful as well.  Or make it chain-able like `where.not`.